### ISSUE TITLE
Pleiads.mra does not work, just gives a "siren" sound

### ISSUE THREAD

**Issue by Videodr0me** [MiSTer-devel/Arcade-Capitol_MiSTer#15]
Pleiads (Centuri).mra works fine, but Pleiads.mra which should be the original version by thekan just gives a blank screen with a siren sound. Maybe some rom is missing or wrong so that some self-check routine fails and gives off a warning sound. 

Maybe its because of the different sets (220 vs 218)? Maybe both should be working with the same set to avoid this?

**Comment by goofyseeker311** [comment]
yep, the plain pleiads, not the centauri/other, will not work properly. mame v0.263 rom.
